.github/workflows/auto-updates.yml

```diff
@@ -86,34 +86,15 @@ jobs:
       - name: Convert to dictionaries
         run: python -Wd -m wikidict ${{ matrix.locale }} --convert
 
-      - name: Upload the dictionary (DictFile)
-        uses: svenstaro/upload-release-action@v2
-        with:
-          repo_token: ${{ secrets.TOKEN }}
-          file: data/${{ matrix.locale }}/dict-${{ matrix.locale }}-${{ matrix.locale }}*.df.bz2
-          tag: ${{ matrix.locale }}
-          overwrite: true
-          file_glob: true
-
-      - name: Upload the dictionary (Kobo)
-        uses: svenstaro/upload-release-action@v2
-        with:
-          repo_token: ${{ secrets.TOKEN }}
-          file: data/${{ matrix.locale }}/dicthtml-${{ matrix.locale }}-${{ matrix.locale }}*.zip
-          tag: ${{ matrix.locale }}
-          overwrite: true
-          file_glob: true
-
-      - name: Upload the dictionary (StarDict)
-        uses: svenstaro/upload-release-action@v2
+      - name: Generate the release description
+        run: python -Wd -m wikidict ${{ matrix.locale }} --release > release.txt
```

Comment:
I didn't ry to hard to skip such step. Maybe can we use the script output directly in the next step :thinking: